blocks/blogs/blogs.js

```diff
@@ -0,0 +1,125 @@
+import { callMockBlogAPI } from '../../scripts/mockapi.js';
+import { readBlockConfig } from '../../scripts/aem.js';
+import { addIcon } from '../../scripts/blocks-utils.js';
```

Comment:
we should use `decorateIcons` from aem.js lib , instead of creating new.

Reply:
in `decorateIcons` selector works on `span.icon`, I have to change implementation based on that selector. Let me try if I can match it that way

Reply:
I used the same in trending news. you can check that https://github.com/aemsites/icicidirect/pull/61/files#diff-cae590c5182580caec0e4af1ee35a12edd5944ccdb31c3d2d5daa33d0839ddb4R52-R53
then line #61 calling `decorateIcons` with element.
and I think we both need time icon and i have also aded in the repo, we can settle on one file, let me know.